linux_mode/scripts/fuzzbpkt.py

```diff
@@ -0,0 +1,268 @@
+# imports
```

Comment:
I feel like we should merge those Python script inside a single file. I don't think the way the logic is split makes it easier to understand / maintain, thoughts?